src/entities/Singleton.java

```diff
@@ -183,4 +186,17 @@ public void setAvailability(boolean availability) {
     }
 
 
+    @Override
+    public JSONObject jsonify() {
+        JSONObject final_data = new JSONObject();
+        final_data.put("id", this.id);
+        final_data.put("price", this.price);
+        final_data.put("name", this.name);
+        final_data.put("description", this.description);
+        final_data.put("allowedAddonTypes", this.allowedAddonTypes);
```

Comment:
These need to be jsonified

Reply:
You cant directly put objects into a json object, use the jsonify method

Reply:
<img width="576" alt="Screenshot 2021-12-02 at 12 15 01 PM" src="https://user-images.githubusercontent.com/73967834/144470561-fd7b3ad4-11c0-42ff-a4ba-a868f89bb102.png">

Reply:
Is this what you mean?